apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -1904,13 +1904,17 @@ export const local_development: NavMenuConstant = {
       items: [
         { name: 'Getting started', url: '/guides/local-development/overview' },
         {
-          name: 'Managing config and secrets',
-          url: '/guides/local-development/managing-config',
+          name: 'Declaring database schemas',
```

Comment:
```suggestion
          name: 'Declarative database schemas',
```

I suggest we use this language (for SEO and searchability